pom.xml

```diff
@@ -74,6 +74,7 @@
         <nimbus-jose-jwt.version>4.41.1</nimbus-jose-jwt.version><!-- Mess in hdfs transitive deps -->
         <okhttp.version>3.14.6</okhttp.version><!-- keep in sync with okio -->
         <okio.version>1.17.2</okio.version><!-- keep in sync with okhttp -->
+        <optaplanner.version>7.42.0.Final</optaplanner.version>
```

Comment:
Shouldn't we use the latest 7.43.1.Final release?

7.42.0.Final is aligned with Quarkus 1.5.2.Final, which is quite old now.

7.43.1.Final at least aligns with Quarkus 1.7.x.